src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Tokens/SamlAuthorizationDecisionClaimResource.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.Serialization;
+
+namespace CoreWCF.IdentityModel.Tokens
+{
+    [DataContract]
+    internal class SamlAuthorizationDecisionClaimResource
```

Comment:
Same as for SamlAuthenticationClaimResource, add the namespace to the DataContract attribute and add the original field naming to the DataMember attributes. Also this should probably be public.